crossfit/backend/torch/model.py

```diff
@@ -41,3 +58,36 @@ def estimate_memory(self, max_num_tokens: int, batch_size: int) -> int:
 
     def max_seq_length(self) -> int:
         raise NotImplementedError()
+
+    def get_model_output(self, all_outputs_ls, index, loader, pred_output_col) -> cudf.DataFrame:
+        from crossfit.backend.torch.loader import SortedSeqLoader
```

Comment:
Sorry missed this earlier, we should move the import up.

Reply:
That was creating cyclic import problem.

Reply:
Just add a comment here.

Reply:
Sure.